src/pages/CreatePool/index.tsx

```diff
@@ -389,7 +389,9 @@ export default function CreatePool(): React.ReactElement {
                   color="secondary"
                   fullWidth
                   exclusive
-                  onChange={(event, value: AssetType) => setAssetType(value)}
+                  onChange={(event, value: AssetType) => {
+                    if (value !== null) setAssetType(value)
```

Comment:
Is this check similar to the one above? If yes, then we should keep it consistent.

Reply:
asset type is a number so it has to explicitly check for null rather than falsy as 0 will eval as falsy

Reply:
are you referring to assetType?